templates/users.ldif.j2

```diff
@@ -11,12 +11,19 @@ memberUid: {{ user }}
 
 {% for user, config in users.items() %}
 dn: uid={{ user }},ou=Users,{{ ldap_domain }}
-cn: {{ user }}
-sn: {{ user }}
+{% if 'fullname' in config %}
+cn: {{ config['fullname'] }}
```

Comment:
i think the CN should be the same as the uid in the dn.  im far from an ldap expert but i think this would normally be formated as 

```jinja
dn: uid={{ config['fullname'] }} {{ config['surname'] }},ou=Users,{{ ldap_domain }}
cn: {{ config['fullname'] }} {{ config['surname'] }}
```

however as we are creating unix accounts i think its better to keep the cn and the dn uid as the user value